pom.xml

```diff
@@ -176,4 +220,148 @@
+                            <execution>
+                                <id>attach-sources</id>
+                                <goals>
+                                    <goal>jar</goal>
+                                </goals>
+                                <configuration>
+                                    <attach>true</attach>
+                                </configuration>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.codehaus.mojo</groupId>
+                        <artifactId>flatten-maven-plugin</artifactId>
+                        <version>1.5.0</version>
+                        <configuration>
+                            <flattenMode>oss</flattenMode>
+                            <updatePomFile>true</updatePomFile>
+                            <outputDirectory>${project.build.directory}</outputDirectory>
+                            <pomElements>
+                                <repositories>remove</repositories>
+                            </pomElements>
+                        </configuration>
+                        <executions>
+                            <execution>
+                                <id>flatten</id>
+                                <phase>process-resources</phase>
+                                <goals>
+                                    <goal>flatten</goal>
+                                </goals>
+                            </execution>
+                            <execution>
+                                <id>flatten.clean</id>
+                                <phase>clean</phase>
+                                <goals>
+                                    <goal>clean</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                    </plugin>
+                    <plugin>
+                        <groupId>org.jreleaser</groupId>
+                        <artifactId>jreleaser-maven-plugin</artifactId>
+                        <inherited>false</inherited>
+                        <configuration>
+                            <jreleaser>
+                                <release>
+                                    <github>
+                                        <owner>mcollovati</owner>
+                                        <name>quarkus-hilla</name>
+                                        <changelog>
+                                            <formatted>ALWAYS</formatted>
+                                            <preset>conventional-commits</preset>
+                                            <skipMergeCommits>false</skipMergeCommits>
+                                            <hide>
+                                                <categories>
+                                                    <category>merge</category>
```

Comment:
Is this also hiding a merge or only if it starts with `merge:`?

Reply:
I got it that it skips commits labelled with `merge`

Reply:
No, I was wrong. It skips merge commits, so I suppose it will remove from changelog lines like " Merge branch 'main' into chore/release-configuration"

Reply:
Following the docs: https://jreleaser.org/guide/latest/reference/release/changelog.html
It seems, that the categories must be defined by self independently from the hide. There are the matching labels defined, so I suppose this hide config does nothing atm.

Reply:
I also found the issue for the skipMergeCommits flag: https://github.com/jreleaser/jreleaser/issues/858
It's effectively a shorthand to skip creation of categories and labels to hide merge commits. In our case I think we can activate it, because squashed PR's are dropping the merge commit to look like a single commit.

Reply:
> It seems, that the categories must be defined by self independently from the hide. There are the matching labels defined, so I suppose this hide config does nothing atm

Shouldn't the preset handle this? https://jreleaser.org/guide/latest/reference/release/changelog.html#_conventional_commits

Reply:
Additional the PR: https://github.com/jreleaser/jreleaser/pull/867
It mentions to rely on the parents of a commit to identify a merge commit, we have to investigate, if a squash still has more than one parent.

Reply:
I mean, we have the `Conventional Commits` preset configures, and we are hiding some of the categories defined by the preset

Reply:
> I mean, we have the `Conventional Commits` preset configures, and we are hiding some of the categories defined by the preset

You're right. I didn't know what the preset does.